src/Microsoft.AspNet.OData.Shared/Formatter/ODataStreamMediaTypeMapping.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.OData.Formatter
+{
+    /// <summary>
+    /// Media type mapping that associates requests with stream property.
+    /// </summary>
+    public partial class ODataStreamMediaTypeMapping
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ODataStreamMediaTypeMapping"/> class.
+        /// </summary>
+        public ODataStreamMediaTypeMapping()
```

Comment:
>ODataStreamMediaTypeMapping [](start = 15, length = 27)

If I know the media type is something other than application/octet-stream, can I specify that somewhere?

Reply:
So far, customer can create the Formatter and set the content type.
Later, we'd figure out a solution for customer to set up the content type.

---
In reply to: [574808287](https://github.com/OData/WebApi/pull/2401#discussion_r574808287) [](ancestors = 574808287)